elasticizer/__main__.py

```diff
@@ -4,33 +4,41 @@
 import random
 from elasticizer import Load
 import collections
+import configargparse
+
 
 def buildArgParser():
```

Comment:
Yeah, that's a good call, I was thinking of doing a separate Pep8 PR because the code is tripping a bunch of violations, but I can fix here if preferred.